spec/defines/seboolean_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'spec_helper'
```

Comment:
This is one the right track. Check out spec/defines/package_spec.rb and use that as a guide